File: javascript/5.js
function isPalindrome(x) {
  let numberString = x.toString();
  let reversedString = numberString.split('').reverse().join('');
  return numberString === reversedString;
}

console.log("\n\n\n" + isPalindrome(121));
console.log(isPalindrome(212));
console.log(isPalindrome(98) + "\n\n\n");
  

File: python/5.py
def is_palindrome(x):
   number_string = str(x)
   reversed_string = number_string[::-1]
   return number_string == reversed_string

print("\n\n\n" + str(is_palindrome(121)))
print(is_palindrome(212))
print(is_palindrome(98) + "\n\n\n")
